build.rs

```diff
@@ -1,10 +1,13 @@
 fn main() {
+    println!("cargo:rerun-if-env-changed=CUSTOM_LIBFUZZER_PATH");
     if let Ok(custom) = ::std::env::var("CUSTOM_LIBFUZZER_PATH") {
         let custom_lib_path = ::std::path::PathBuf::from(&custom);
```

Comment:
Adding a `println!("cargo:rerun-if-changed={custom}");` here might be useful :)

Reply:
Fair point -- added!